docs/api.md

```diff
@@ -2464,9 +2464,9 @@ expect(await tweetHandle.$eval('.retweets', node => node.innerText)).toBe('10');
 
 #### elementHandle.$x(expression)
 - `expression` <[string]> Expression to [evaluate](https://developer.mozilla.org/en-US/docs/Web/API/Document/evaluate).
-- returns: <[Promise]<?[ElementHandle]>> Promise which resolves to ElementHandle pointing to the frame element.
+- returns: <[Promise]<[Array]<[ElementHandle]>>> Promise which resolves to ElementHandle pointing to the frame element.
```

Comment:
can you please also remove the confusing description for the return value? It's just a copy-paste leftover.

Reply:
sure, NP